tests/aead_tests.rs

```diff
@@ -34,6 +34,8 @@
 
 extern crate ring;
 
+mod constants;
```

Comment:
Instead of creating a new module, let's just put them in the `ring::test` module, so they will also be available for unit tests, in addition to integration tests.